### ISSUE TITLE
Don't activate easyGestures inside a text field

### ISSUE THREAD

**Issue by ris58h** [ngdeleito/easyGestures#17]
easyGesture activates when I try to select a text inside a text field using `shift+left_click`.

easyGestures 6.6
Firefox 105.0

**Comment by ngdeleito** [comment]
Yes, indeed, I can reproduce the problem. It's a bit of an unfortunate key combination :( ... I guess that if you click first (and keep the left button pressed) and then press shift to select, the pie menu is not activated... Really don't know how to solve this; this is clearly a conflicting key combination...

**Comment by ris58h** [comment]
Could we detect that textfield is focused/active and prevent the pie menu from activation?

**Comment by ngdeleito** [comment]
Perhaps, but you might want to activate the contextual menu instead... It is really an unfortunate key combination...
